pkg/codegen/internal/test/testdata/aws-get-policy-document-pp/aws-get-policy-document.pp

```diff
@@ -0,0 +1,53 @@
+data "aws_iam_policy_document" "example" {
```

Comment:
This likely needs to be an `invoke`, see https://github.com/pulumi/pulumi/blob/ae247f8f2e5cb627e1ba92e2e603daf166ef37bf/pkg/codegen/internal/test/testdata/aws-fargate-pp/aws-fargate.pp#L5 for example

Reply:
Ah, for my understanding this is because the `.pp` files are in PCL?

Reply:
Yes, I think pp = pulumi program.